src/RadStreamingY/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+if (AMReX_SPACEDIM EQUAL 2)
```

Comment:
We don't have any CI tests set up for AMReX_SPACEDIM=2. So either you should add them, or this should should compile and run when AMReX_SPACEDIM=3.